docs/DeveloperGuide.md

```diff
@@ -207,6 +207,31 @@ The following activity diagram summarizes what happens when a teacher executes a
     2. Only can set the class to a date at most 1 week away.
   
 
+### Schedule List Feature
+This feature allows the user to be able to view a schedule on the right hand side of the panel.
+
+#### Implementation Details
+In order to update the `UI` from original view of a single panel to the right panel, there was a
+need to modify the existing `UI`. As a result, `MainWindow.fxml` had to be modified.
+
+Since there is a need for the schedule list to be a constant panel, which is separate from the `PersonListPanel`,
+a `ScheduleListPanel` under `UI` had to be created along with a `ScheduleListCard` class also under `UI`. The
+`ScheduleListCard` will contain more specific information than `PersonCard`. 
```

Comment:
it should be `PersonCard` contains more specific information than `ScheduleListCard` like `MoneyOwed`, `MoneyPaid`, `RatesPerClass` right